### PR TITLE
added content_css_cors setting documentation

### PR DESCRIPTION
**Pull request by fyrkant**

### CODE REVIEW

**Comment by shikhanansi** [comment]
@fyrkant what is the status of this branch? Is the content going in the 5.0 Beta docs?

**Comment by fyrkant** [comment]
Yes it should be a part of the 5 docs.

**Comment by shikhanansi** [comment]
@EkimChau @blackcathikari please review this PR.

File: _includes/configuration/content-css-cors.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+## content_css_cors
+When setting the `content_css_cors` setting to `true` the editor will add a `crossorigin="anonymous"` attribute to the link tags that the StyleSheetLoader uses when loading the `content_css`, allowing you to host the `content_css` on a different server than the editor itself.
```
Comment (blackcathikari) [inline _includes/configuration/content-css-cors.md]: This is a little hard to parse - could we split it into two sentences or something? 

Maybe: "When `content_css_cors` is set to `true` the editor will add a `crossorigin="anonymous"` attribute to the link tags that the StyleSheetLoader uses when loading the `content_css`. This allows you to host the `content_css` on a different server than the editor itself."
